acmw-db-app/pages/signin.js

```diff
@@ -27,7 +27,7 @@ const SignIn = () => {
               label="Password"
               type="password"
             />
-            <a className={styles.smol} href="https://crouton.net/">Forgot username or password?</a>
+            <a className={styles.smol} href="../reset">Forgot username or password?</a>
```

Comment:
I'm not sure if we should use it but React has a Link tag that seems handy https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function

Reply:
I'm just going to leave it how it is because they do the same thing.